### ISSUE TITLE
Discord? Slack? A place to chat?

### ISSUE THREAD

**Issue by Uzlopak** [node-oauth/node-oauth2-server#78]
Can we have a place to chat? 
Slack or Discord would be cool. Then I could annoy you constantly with OAuth2-related stuff.

**Comment by HappyZombies** [comment]
I'm down, but I'm not volunteering to be a mod/admin or whatever cuz idk how to run those things. Also is Gitter a thing or do people not really use that?

**Comment by Uzlopak** [comment]
I am OK with gitter.

**Comment by jankapunkt** [comment]
I think gitter is ok. I also use discord.

**Comment by jorenvandeweyer** [comment]
I'd prefer Discord or Slack to follow up on discussions.

**Comment by HappyZombies** [comment]
I will say though that we should use these other tools as an extra thing to look up questions, not solely use discord or Gitter or whatever. 

I can’t stand when I read on some projects “for questions, check out our discord”. Which I interpret as “looking up questions isn’t a thing”. Not everyone is gonna wanna make a account just to ask something, some people search in GitHub issues for questions and answers. 

Basically, if we move over to discord, it shouldn't be our main tool for questions.

**Comment by jankapunkt** [comment]
I'd rather use them for longer discussions to keep issues on point, especially when topics are broad. Questions should be asked in these issues sections.

**Comment by jorenvandeweyer** [comment]
I went a head and created a Discord server.

https://discord.gg/4HTUAcTvqV
